shuup/xtheme/urls.py

```diff
@@ -10,9 +10,37 @@
 from shuup.xtheme.views.command import command_dispatch
 from shuup.xtheme.views.editor import EditorView
 from shuup.xtheme.views.extra import extra_view_dispatch
+from shuup.xtheme.views.plugins import (
+    get_category_products_highlight, get_product_cross_sell_highlight,
+    get_product_highlight, get_prouduct_selections_highlight
+)
 
 urlpatterns = [
     url(r"^xtheme/editor/$", EditorView.as_view(), name="xtheme_editor"),
     url(r"^xtheme/(?P<view>.+)/*$", extra_view_dispatch, name="xtheme_extra_view"),
     url(r"^xtheme/$", command_dispatch, name="xtheme"),
+    url(
```

Comment:
I would be nice to put these async plugins into a different app, what do you think? looks like this is polluting the xtheme base app.